apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerParameterTests.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
For some reason a lot of these first lines are being shown as changed in GitHub, although there's no visible change, maybe an end of line change?

Reply:
It's almost certainly BOM removal, which is fine.